components/post_view/post_info/post_info.tsx

```diff
@@ -171,7 +181,7 @@ export default class PostInfo extends React.PureComponent {
         return this.refs.dotMenu;
     };
 
-    buildOptions = (post, isSystemMessage, fromAutoResponder) => {
+    buildOptions = (post : any, isSystemMessage : boolean, fromAutoResponder : any) => {
```

Comment:
`fromAutoResponder` should be of type `boolean`

Reply:
Same here with `Post` type

Reply:
Done for the spacing @devinbinnie .
Not done yet for the `Post` type.